packages/utilities/gel-foundations/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/gel-foundations",
-  "version": "6.1.3",
+  "version": "8.0.0",
```

Comment:
I'd say this should be a minor bump as this is a non-breaking change, also why jump 2 major versions anyway? See for some guidance: https://github.com/bbc/psammead/pull/4545#discussion_r708161813

Reply:
Due to the nature of this work, where we've split it into 3 different PR's, we've tried to plan ahead as the order we wanted to merge was Tamil, Bengali and then this PR hence the weird bumping. Next time it will probably be easier to keep the work under 1 PR, less bumping headaches which we didn't anticipate and each font was in a different issue